src/systemctl/systemctl.c

```diff
@@ -8211,7 +8227,7 @@ int main(int argc, char*argv[]) {
         if (r <= 0)
                 goto finish;
 
-        if (arg_action != ACTION_SYSTEMCTL && running_in_chroot() > 0) {
+        if (arg_action != ACTION_SYSTEMCTL && running_in_chroot() > 0 && !arg_allow_chroot) {
```

Comment:
there are a couple of more cases where we check for chroots in systemctl.c, what about those? There's also the VERB_NOCHROOT logic used all over the place, what about that?

Maybe the env var check should be added to running_in_chroot() itself, so that it applies everywhere? (and then probably be renamed to `$SYSTEMD_IGNORE_CHROOT` or so?